website/payments/models.py

```diff
@@ -92,6 +116,74 @@ def __str__(self):
         return _("Payment of {amount}").format(amount=self.amount)
 
 
+def _default_batch_description():
+    now = timezone.now()
+    return f"Thalia Pay payments for {now.year}-{now.month}"
+
+
+class Batch(models.Model):
+    """
+    Describes a batch of payments for export
+    """
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    processed = models.BooleanField(verbose_name=_("processing status"), default=False,)
+
+    processing_date = models.DateTimeField(
+        verbose_name=_("processing date"), blank=True, null=True,
+    )
+
+    description = models.TextField(
+        verbose_name=_("description of batch"), default=_default_batch_description,
```

Comment:
```suggestion
        verbose_name=_("description"), default=_default_batch_description,
```
If this is a field called 'description' in a batch then it's pretty logical right? Just needs the translation.